docs/pages/database-access/guides/mysql-cloudsql.mdx

```diff
@@ -24,12 +24,47 @@ description: How to configure Teleport database access with GCP Cloud SQL MySQL.
   Service
 - (!docs/pages/includes/tctl.mdx!)
 
-## Step 1/5. Create a service account for the Teleport Database Service
+## Step 1/7. Enable Cloud SQL IAM authentication
 
-Teleport uses one-time passwords to authenticate with Cloud SQL MySQL. To be
-able to authenticate with a database instance, Teleport must run as a service
-account that has a few of the "Cloud SQL Admin" role permissions. You can create
-a new service account or modify an existing one to add required permissions.
+Teleport can access MySQL on GCP Cloud SQL as built-in database users with
+passwords or database users with [IAM
+authentication](https://cloud.google.com/sql/docs/mysql/iam-authentication)
+enabled.
+
+Teleport generates a one-time password for built-in database users and updates
```

Comment:
Can we just remove the mention of the OTP from everywhere in the docs?

Reply:
removed.